docs/guides/connecting_to_your_data/database/snowflake.md

```diff
@@ -1,4 +1,162 @@
 ---
 title: How to connect to your data in a Snowflake database
 ---
-This article is a stub.
+import Prerequisites from '../components/prerequisites.jsx'
+import WhereToRunCode from '../components/where_to_run_code.md'
+import NextSteps from '../components/next_steps.md'
+import Congratulations from '../components/congratulations.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect to data in a Snowflake database.
+This will allow you to validate and explore your data.
+
+<Prerequisites>
+
+- Have access to data in a Snowflake database
+
+</Prerequisites>
+
+## Steps
+
+### 1. Choose how to run the code in this guide
+
+<WhereToRunCode />
+
+### 2. Install required dependencies
+
+First, install the necessary dependencies for Great Expectations to connect to your Snowflake database by running the following in your terminal:
+
+```console
+pip install sqlalchemy
+pip install snowflake-connector-python
+pip install snowflake-sqlalchemy
+```
+
+### 3. Add credentials
+
+Great Expectations provides multiple methods of using credentials for accessing databases.
+Options include using an file not checked into source control, environment variables, and using a cloud secret store.
+Please read the article [Credential storage and usage options](../advanced/database_credentials) for instructions on alternatives.
```

Comment:
Let's make lines 38-40 an importable module since it'll be the same for each database article!